serverless/jobs/quickstart.mdx

```diff
@@ -0,0 +1,94 @@
+---
+meta:
+  title: Jobs - Quickstart
+  description: This page shows you how to get started with Scaleway Serverless Jobs
+content:
+  h1: Jobs - Quickstart
+  paragraph: This page shows you how to get started with Scaleway Jobs
+tags: jobs serverless namespace
+dates:
+  validation: 2023-11-07
+  posted: 2023-11-07
+categories:
+  - serverless
+  - jobs
+---
+
+Scaleway's Serverless Jobs allow you to run recurring tasks in the cloud. A Job is composed of a job definition, and some job runs.
+
+This page explains how to create a job definition with the latest Alpine Linux image, how to execute it and delete it.
+
+<Macro id="iam-requirements" />
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com/) 
+  - You have [signed up to the private beta](https://www.scaleway.com/en/betas/#serverless-jobs) and received a confirmation email.
+</Message>
+
+## How to create a job definition
+
+You can create a job either from the Scaleway Container Registry or any other public container registry (Docker Hub, AWS Container registries, GitLab container registry, etc.).
+
+### From the Scaleway Container Registry
```

Comment:
I propose we keep this quick-start as simple as possible, just using an external registry for now, then link to the specific docs on Scaleway registry.